core/server/master/src/main/java/alluxio/master/AlluxioMasterProcess.java

```diff
@@ -280,9 +280,9 @@ protected void startServingRPCServer() {
       GrpcServerBuilder serverBuilder = GrpcServerBuilder.forAddress(mRpcBindAddress,
           ServerConfiguration.global());
 
-      mRPCExecutor =
-          new ForkJoinPool(ServerConfiguration.getInt(
-              PropertyKey.MASTER_RPC_FORKJOIN_POOL_PARALLELISM));
+      mRPCExecutor = Executors.newFixedThreadPool(ServerConfiguration.getInt(
+          PropertyKey.MASTER_RPC_THREAD_POOL_PARALLELISM));
```

Comment:
Does it make sense to supply a thread factory?

Reply:
Maybe? what do we want to do in a custom thread factory?

Reply:
We can name the threads and make them daemon threads.